Examples/ConsoleExample/ConsoleExample.csproj

```diff
@@ -33,6 +34,7 @@
     <DefineConstants>TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>
```

Comment:
It should be OK to leave it. It's determined by the runtime environment when using Any CPU as in Configuration Manager.